### PR TITLE
[IA] Relocate Java Automatic pages to new Zero-code section

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to #4428 by relocating the files
- **Preview**: https://deploy-preview-4430--opentelemetry.netlify.app/docs/zero-code/java/

@svrnm @zeitlinger et all:

- Is this what we had envisioned?
- Should **App server config**, currently under the Java Agent section, be moved up one level?

### Screenshots

The new left-nav:

>

### CODE REVIEW

**Comment by zeitlinger** [comment]
looks great - I'm double checking with the Java SIG to be sure

File: content/en/docs/zero-code/_index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+[Issue #4427]: https://github.com/open-telemetry/opentelemetry.io/issues/4427
+
+{{% /pageinfo %}}
+
+OpenTelemetry [zero-code instrumentation][] is supported for the languages
+listed below in the section index.
+
+If you are using Kubernetes, you can use the [OpenTelemetry Operator for
+Kubernetes][otel-op] to [inject auto-instrumentation libraries][auto] for .NET,
```
Comment (svrnm) [inline content/en/docs/zero-code/_index.md:20]:
```suggestion
Kubernetes][otel-op] to [inject zero-code instrumentations][operator-injection] for .NET,
```

File: content/en/docs/zero-code/_index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Java, Node.js, Python, Go into your application.
+
+[auto]: /docs/kubernetes/operator/automatic/
```
Comment (svrnm) [inline content/en/docs/zero-code/_index.md:23]:
```suggestion
[operator-injection]: /docs/kubernetes/operator/automatic/
```

File: content/en/docs/zero-code/java/_index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# FIXME: ensure version update script targets this file too
+cascade:
+  vers:
+    instrumentation: 2.3.0
+    otel: 1.37.0
+cSpell:ignore: Dotel myapp
+---
+
+Zero-code instrumentation with Java can use a Java agent JAR or Spring Boot
+Start. To learn how to manually instrument your service or app code, see
```
Comment (svrnm) [inline content/en/docs/zero-code/java/_index.md:16]:
```suggestion
Starter. To learn how to manually instrument your service or app code, see
```

File: content/en/docs/zero-code/java/agent/_index.md [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
-weight: 30
 ---
 
-Automatic instrumentation with Java uses a Java agent JAR that can be attached
-to any Java 8+ application. It dynamically injects bytecode to capture telemetry
-from many popular libraries and frameworks. It can be used to capture telemetry
-data at the "edges" of an app or service, such as inbound requests, outbound
-HTTP calls, database calls, and so on. To learn how to manually instrument your
-service or app code, see [Manual instrumentation](../instrumentation).
+Zero-code instrumentation with Java case use a Java agent JAR attached to any
```
Comment (svrnm) [inline content/en/docs/zero-code/java/agent/_index.md:11]:
```suggestion
Zero-code instrumentation with Java uses a Java agent JAR attached to any
```

**Review by svrnm — CHANGES_REQUESTED** [review]
LGTM, thanks for driving this! There are still a few places with "manual/automatic" which we should consider replacing with "code-based" and "zero-code" but I think that's one of the many follow ups we need to do

**Comment by chalin** [comment]
I reapplied your suggested changes because I overwrote them by mistake. PTAL @svrnm

**Review by svrnm — APPROVED** [review]
LGTM!

**Comment by svrnm** [comment]
@open-telemetry/java-approvers @open-telemetry/java-instrumentation-approvers PTAL!

File: content/en/docs/zero-code/_index.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+[Issue #4427]: https://github.com/open-telemetry/opentelemetry.io/issues/4427
+
+{{% /pageinfo %}}
+
+OpenTelemetry [zero-code instrumentation][] is supported for the languages
+listed below in the section index.
+
+If you are using Kubernetes, you can use the [OpenTelemetry Operator for
+Kubernetes][otel-op] to [inject zero-code instrumentation] for .NET, Java,
+Node.js, Python, Go into your application.
```
Comment (cartermp) [inline content/en/docs/zero-code/_index.md:21]:
```suggestion
Node.js, Python, or Go into your application.
```

**Review by cartermp — APPROVED** [review]

**Review by theletterf — APPROVED** [review]

File: content/en/docs/zero-code/java/_index.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+title: Java zero-code instrumentation
+linkTitle: Java
+aliases:
+  - /docs/java/automatic_instrumentation
+  - /docs/languages/java/automatic_instrumentation
+# FIXME: ensure version update script targets this file too
+cascade:
+  vers:
+    instrumentation: 2.3.0
```
Comment (zeitlinger) [inline content/en/docs/zero-code/java/_index.md:10]:
```suggestion
    instrumentation: 2.4.0
```

File: content/en/docs/zero-code/java/_index.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    otel: 1.37.0
```
Comment (zeitlinger) [inline content/en/docs/zero-code/java/_index.md:11]:
```suggestion
    otel: 1.38.0
```

Comment (svrnm) [reply content/en/docs/zero-code/java/_index.md:11]: We might need to fix some automation here as well post merge

**Comment by zeitlinger** [comment]
I think we need redirects from the old pages

**Comment by svrnm** [comment]
> I think we need redirects from the old pages

From what I see in the code @chalin took care of that, any specific redirects you are missing?

**Comment by zeitlinger** [comment]
> > I think we need redirects from the old pages
> 
> From what I see in the code @chalin took care of that, any specific redirects you are missing?

no, just missed it...

**Review by svrnm — APPROVED** [review]

**Comment by cartermp** [comment]
@chalin unfortunately it seems there are some conflicts now

**Comment by svrnm** [comment]
> @chalin unfortunately it seems there are some conflicts now

@chalin is out of office this week as far as I know, I can take a look

**Comment by zeitlinger** [comment]
thanks a lot for the help!

File: content/en/docs/zero-code/java/_index.md (hunk 7 above)
Comment (chalin) [reply content/en/docs/zero-code/java/_index.md]: Yes, this is already a task marked in #4428.

**Pull request merged**